README.md

```diff
@@ -331,26 +326,30 @@ back to full health.
 Learn more about this essential process in the
 [conflicts documentation](https://github.com/cloudant/sync-android/blob/master/doc/conflicts.md).
 
-## Known Issues
+## Issues
+
+Before opening a new issue please consider the following:
+* Only the latest release is supported. If at all possible please try to reproduce the issue using
+the latest version.
+* Please check the [existing issues](https://github.com/cloudant/sync-android/issues)
+to see if the problem has already been reported. Note that the default search
+includes only open issues, but it may already have been closed.
+* Cloudant customers should contact Cloudant support for urgent issues.
+* When opening a new issue [here in github](../../issues) please complete the template fully.
 
-Some users on certain older versions of Android have reported the
```

Comment:
Are we confident this is no longer an issue?

Reply:
@ricellis I removed it because of the `Only the latest release is supported` writing in other libraries.  @tomblench do you know if this issue is still happening? I'm fine to add it back under this section.

Reply:
Yes, we only supported the latest release of sync-android, but I thought this was relevant to older versions of Android itself, rather than sync-android.

Reply:
@ricellis I added it back in 4ad4509 as it might still be relevant.